.github/workflows/rust.yml

```diff
@@ -47,10 +45,15 @@ jobs:
         echo ::set-output name=package_list::$(colcon list --names-only)
 
     - name: Setup ROS environment
-      uses: ros-tooling/setup-ros@v0.3
+      uses: ros-tooling/setup-ros@v0.6
       with:
+        use-ros2-testing: true
```

Comment:
Can you remove this? I'll prepare a PR that will enable `use-ros2-testing` and will fix the build issue with `rolling`, but this change is not necessary for removing EOL'd distros.

Reply:
Done